tests/appsec/iast/test_grpc_iast.py

```diff
@@ -148,3 +150,19 @@ class MyUserDict(UserDict):
             mutable_mapping = MyUserDict(original_dict)
 
             _custom_protobuf_getattribute(mutable_mapping, "data")
+
+    def test_address_server_data(self):
+        with override_env({"DD_IAST_ENABLED": "True"}):
+            with override_global_config(dict(_asm_enabled=True)):
+                with override_config("grpc", dict(service_name="myclientsvc")):
+                    with override_config("grpc_server", dict(service_name="myserversvc")):
+                        with mock.patch("ddtrace.appsec._asm_request_context.set_waf_address") as mock_set_waf_addr:
+                            channel1 = grpc.insecure_channel("localhost:%d" % (_GRPC_PORT))
+                            stub1 = HelloStub(channel1)
+                            res = stub1.SayHello(HelloRequest(name="test"))
+                            assert hasattr(res, "message")
+                            mock_set_waf_addr.assert_any_call(SPAN_DATA_NAMES.GRPC_SERVER_RESPONSE_MESSAGE, mock.ANY)
+                            mock_set_waf_addr.assert_any_call(SPAN_DATA_NAMES.GRPC_SERVER_REQUEST_METADATA, mock.ANY)
```

Comment:
Are we OK with just checking for mock.ANY here? Just a sanity check since you specified a path in the last call for the `GRPC_SERVER_METHOD` one

Reply:
Yeah it's a big dict with some changing things like IP, trace-id, user-agent, etc.